apps/api/src/server/repositories/folder.repository.js

```diff
@@ -8,3 +8,65 @@ import { databaseConnector } from '../utils/database-connector.js';
 export const getByCaseId = (caseId) => {
 	return databaseConnector.folder.findMany({ where: { caseId } });
 };
+
+export const createFolders = () => {
+	return databaseConnector.folder.create({
+		data: [
```

Comment:
Could we break out this array into a variable elsewhere in this file? This is for readability purposes